src/mame/includes/taito_l.h

```diff
@@ -28,30 +30,23 @@ class taitol_state : public driver_device
 	u8 *       m_shared_ram;
 
 	/* video-related */
-	tilemap_t *m_bg18_tilemap;
-	tilemap_t *m_bg19_tilemap;
-	tilemap_t *m_ch1a_tilemap;
-	u8 m_buff_spriteram[SPRITERAM_SIZE];
+	tilemap_t *m_bg_tilemap[0];
+	tilemap_t *m_bg_tilemap[1];
```

Comment:
you're redeclaring stuff m_bg_tilemap[0] here, that you have declared the line above